tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
```

Comment:
Also, this value should always be the same as the maxRecordCount we return as part of the FeatureServiceLayerDetailView and FeatureServiceDetailView. Should probably lift this out to a constant.